application/dash_application/views/pre_exome.py

```diff
@@ -392,36 +413,36 @@ def updatePressed(click,
             passedfilter):
 
         # Apply get selected runs
-        data = bamqc[bamqc[bamqc_cols.Run].isin(runs)]
-        data[bamqc_cols.GroupID] = data[bamqc_cols.GroupID].fillna("")
+        data = bamqc[bamqc[BAMQC_COL.Run].isin(runs)]
+        data[BAMQC_COL.GroupID] = data[BAMQC_COL.GroupID].fillna("")
```

Comment:
The `get_pinery_samples` function from earlier takes care of the `fillna` for GroupID